source/MaterialXShaderGen/ShaderGenerators/Glsl/SurfaceShaderGlsl.cpp

```diff
@@ -1,26 +1,26 @@
-#include <MaterialXShaderGen/ShaderGenerators/Glsl/AdskSurfaceGlsl.h>
+#include <MaterialXShaderGen/ShaderGenerators/Glsl/SurfaceShaderGlsl.h>
 #include <MaterialXShaderGen/ShaderGenerators/Glsl/GlslShaderGenerator.h>
 
 namespace MaterialX
 {
 
-SgImplementationPtr AdskSurfaceGlsl::creator()
+SgImplementationPtr SurfaceShaderGlsl::creator()
```

Comment:
Changes this to an implementation class that can be used by any surface shader that has source code but needs to add shader variables and have support for isTransparent, etc.